Application/Telegram/TelegramCommunicationService.cs

```diff
@@ -9,15 +11,35 @@
 using System.Threading.Tasks;
 using Telegram.Bot;
 using Telegram.Bot.Types;
+using Telegram.Bot.Types.Enums;
+using Telegram.Bot.Types.ReplyMarkups;
 
 namespace Application.TelegramBot
 {
     public class TelegramCommunicationService : ITelegramCommunicationService
     {
         private readonly TelegramBotClient _telegraBotClient;
-        public TelegramCommunicationService(IConfiguration configuration)
+        private readonly IMediator _mediator;
+        public TelegramCommunicationService(IConfiguration configuration, IMediator mediator)
         {
             _telegraBotClient = new TelegramBotClient(configuration["Token"]);
+            _mediator   = mediator;
+        }
+
+        public async Task Execute(Update update)
+        {
+            if (update.Message != null && update.Message.Text.Contains("/start"))
```

Comment:
you can move check for not null to the beginning in 1 place, so you will not need to check in each row.